source/stylist/style.py

```diff
@@ -65,15 +65,18 @@ def _all_subclasses(cls):
 
 
 def read_style(rule_file: Path, style_name: Optional[str] = None) -> Style:
+    PREFIX = 'style.'
+
     configuration = configparser.ConfigParser()
     try:
         with rule_file.open('rt') as handle:
             configuration.read_file(handle)
     except configparser.MissingSectionHeaderError:
         pass  # It is not an error to have an empty configuration file.
 
-    available_styles = [section[6:] for section in configuration.sections()
-                        if section.startswith('style.')]
+    available_styles = [section[len(PREFIX):]
+                        for section in configuration.sections()
+                        if section.startswith(PREFIX)]
 
```

Comment:
I definitely prefer this over the hard-coded index, it may never come up but I think it's better to be prepared for another prefix.